cfep-10.md

```diff
@@ -0,0 +1,59 @@
+
+<table>
+<tr><td> Title </td><td> Feedstock Statuses </td>
+<tr><td> Status </td><td> Proposed </td></tr>
+<tr><td> Author(s) </td><td> Christopher J. "CJ" Wright &lt;cjwright4242@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> Sept 29, 2019</td></tr>
+<tr><td> Discussion </td><td> link to the PR where the CFEP is being discussed, NA is circulated initially </td></tr>
+<tr><td> Implementation </td><td> link to the PR for the implementation, NA if not availble </td></tr>
+</table>
+
+## Abstract
+
+This document proposes the creation of four new statuses for feedstocks that denote the	
+standing of the feedstock and transistions between the standings.
+These statuses will help us triage feedstocks and identify which feedstocks need
+help, maintainers, or archiving.
+
+## Motivation
+
+It can be difficult for maintainers and would be maintainers to identify which feedstocks could use their
+help.
+Additionally, it is difficult for core to know which feedstocks need additional help or archiving.
+Finally, users of CF packages may find it difficult to understand which packages are dorment.
+This proposal aims to expose this information more clearly.
+
+## Specification
+
+This proposal creates four new statuses
+
+### In Perfect Standing
+The feedstock is actively maintained, new versions of the packages are being created as needed, autotick bot PRs are addressed 
+(merged, fixed, or closed) promptly. Less than 3 bot PRs are open at once.
```

Comment:
```suggestion
(merged, fixed, or closed) promptly. Less than 3 PRs are open at once.
```

Reply:
Using a hard line like this actively encourages maintainers close user-contributed work-in-progress PRs. Is that a goal? Bot PRs represent tasks that should be done in a timely manner, user PRs may or may not be. I've seen a number of repos with stalled "build on Windows" or "split outputs" PRs outstanding for a long time, where their incompleteness is not an indication of the repo being unmaintained.

While I think a simple numerical metric for open bot PRs is valid, it's harder to have a good metric for user PRs (just counting open doesn't seem good to me, but e.g. open and passing tests without comment since last commit for a length of time...)

Reply:
I'd leave this statement as "Less than 3 bot PRs are open at once" and let community PRs be more vaguely addressed by the previous statement, relying on human judgment instead of bots to enforce "addressed promptly".